ddcz/templatetags/datetime.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+from math import floor
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def timestamp_to_difference(seconds):
```

Comment:
IMHO this whole thing would be better done if you'd just convert to datetime, because than timedelta do validation for you

```
In [1]: from datetime import datetime, timedelta

In [3]: reg_time = datetime.now() - timedelta(days=7)

In [4]: diff = datetime.now() - reg_time

In [7]: print(f"{diff.days}d")
7d
```

Reply:
I am not exactly sure how this will help since in the database is not datetime but unix timestamp. Basically I'd be converting it one way, working, then rendering back. Seems to me that just `if` commands are much faster than creating objects. What validation do I need here?